cmd/juju/cloud/updatecredential.go

```diff
@@ -131,8 +132,25 @@ func (c *updateCredentialCommand) Run(ctx *cmd.Context) error {
 	}
 	defer client.Close()
 
-	if err := client.UpdateCredential(credentialTag, *credToUpdate); err != nil {
-		return err
+	models, err := client.UpdateCredentialsCheckModels(credentialTag, *credToUpdate)
+	if err != nil {
+		ctx.Infof("Could not update credential %q for user %q on cloud %q: %v", c.credential, accountDetails.User, c.cloud, err)
+	}
+	for _, m := range models {
+		if len(m.Errors) == 0 {
+			ctx.Infof("Model %q is visible.", m.ModelName)
```

Comment:
Maybe this would be clearer as "Model "blah" is accessible with updated credential."?

Reply:
If we have even 10 models, this message will become chatting. I was looking for something more laconic and straight to the point :D